cpp/src/strings/convert/convert_floats.cu

```diff
@@ -351,31 +351,36 @@ struct ftos_converter {
 };
 
 template <typename FloatType>
-struct float_to_string_size_fn {
-  column_device_view d_column;
+struct from_floats_fn {
+  column_device_view d_floats;
+  int32_t* d_offsets;
```

Comment:
```suggestion
  size_type* d_offsets;
```
Use `size_type` or `offset_type` here instead. Same for `convert_integers.cu` too.